tests/Microsoft.Identity.Test.LabInfrastructure/LabUserHelper.cs

```diff
@@ -172,5 +179,20 @@ public static string FetchUserPassword(string userLabName)
                 throw new InvalidOperationException("Test setup: cannot get the user password. See inner exception.", e);
             }
         }
+
+        public static Task RevokeUserSessionAsync(LabUser user)
+        {
+            return  s_labService.GetLabResponseAsync(Path.Combine(LabApiConstants.LabUserEndPoint, "revokeusersession", user.Upn));
```

Comment:
what command does the lab execute to revoke the session?

Reply:
It's a placeholder for now. Lab behavior isn't implemented.

Reply:
Suggest we implement the lab functionality into service.msidlab.com that already hosts msi helper service. is there a doc on what needs to be done to get this reset functionality added to the lab?

Reply:
No doc, but the placeholder here is basically all we need. Revoke a specific user's session, disable/enable a specific app.